dashboard/modules/reporter/reporter_agent.py

```diff
@@ -177,7 +200,7 @@ def _get_network_stats():
     @staticmethod
     def _get_mem_usage():
         vm = psutil.virtual_memory()
-        return vm.total, vm.available, vm.percent
+        return vm.total, vm.available, vm.percent, vm.used
```

Comment:
this will break frontend again because frontend reference
https://github.com/ray-project/ray/blob/3ffe375a095f1fc2547c2ca6f12663f52c001c53/dashboard/client/src/api.ts#L156

Reply:
good catch, added the 4th number and manually checked that the dashboard is working